src/config.ts

```diff
@@ -0,0 +1,17 @@
+import { makeConfig } from 'cleaner-config'
+import { asArray, asNumber, asObject, asOptional, asString } from 'cleaners'
+
+export const asConfig = asObject({
+  couchDbFullpath: asOptional(
+    asString,
+    'http://username:password@localhost:5984'
+  ),
+  httpPort: asOptional(asNumber, 8008),
+  soloAppIds: asOptional(asArray(asString), undefined),
+  soloPartnerIds: asOptional(asArray(asString), undefined),
```

Comment:
Interestingly, cleaner config wont include undefined fields in the output. Perhaps make the default `() => []`?

Reply:
actually it would need to be `null`